test/benchmark/common/host_guest_bench_runner.sh

```diff
@@ -29,26 +29,39 @@ fi
 # Parameters:
 #   $1: guest command to run on the VM
 #   $2: output file to store the benchmark results
-#   $3: sleep time before running the host command
+#   $3: log file to monitor for the start signal
+#   $4: start signal string
 run_benchmark() {
     local guest_cmd=$1
     local output_file=$2
-    local sleep_time=$3
+    local log_file=$3
+    local start_signal=$4
```

Comment:
Do not call it "start signal". In a bash script, the first thing that comes to one's mind when she sees the word "signal" is POSIX signals.

Reply:
`ready_message` may be better.